config/tekton-dashboard-deployment.yaml

```diff
@@ -1,42 +1,120 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Template
```

Comment:
Hi @Gl4di4torRr this file is currently being used as a Deployment for those applying the changes directly to their local environment, such as Docker Desktop, minikube etc. I'm concerned that changing this from `kind: Deployment` to `kind: Template` is going to break a number of existing users. Would it be possible to provide the Template as a separate config element, perhaps in its own file?

Reply:
@mnuttall maybe there should be a split between defining specific object types and templates? I can created a folder `config/templates` and place my template files there and leave the deployment object alone for the sake of existing users.

Reply:
Thank you!